CONTRIBUTING.md

```diff
@@ -70,21 +74,21 @@ It is recommended that each of your patches fixes one thing. Smaller patches are
 
 ## Pull requests
 
-We accept [github pull requests] (https://github.com/clearcontainers/shim/pulls).
+We accept [github pull requests](https://github.com/clearcontainers/shim/pulls).
 
-Github has a basic introduction to the process [here] (https://help.github.com/articles/using-pull-requests/).
+Github has a basic introduction to the process [here](https://help.github.com/articles/using-pull-requests/).
```

Comment:
Best practice is to avoid the use of [here]. Replace with: Github has a basic introduction to [using pull requests] (https://help.github.com/articles/using-pull-requests/).